legacy/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "publishConfig": {
+    "main": "./legacy/index.cjs",
+    "module": "./legacy/index.js"
+  },
```

Comment:
```suggestion
  "main": "./legacy/src/index.js",
  "publishConfig": {
    "main": "./legacy/index.cjs",
    "module": "./legacy/index.js"
  },
```

Reply:
@dangreen with this config, we can't import charts from `vue-chartjs/legacy`

Reply:
@thabarbados this package.json shouldn't be in published package.

let's restrict package files with "files" field

https://github.com/reactchartjs/react-chartjs-2/blob/master/package.json#L31-L33

Reply:
Thanks, I missed  `Files` field settings. Fixed.